find the sum of the largest sum continuous array.cpp

```diff
@@ -0,0 +1,85 @@
+// PROBLEM :: find the sum of the largest sum continuous array.
+
+#include <bits/stdc++.h>
+using namespace std;
+
+// METHOD 1 : USING KADANE'S ALGORITHM
+// DOESN'T WORK IF ALL THE ELEMENTS ARE NEGATIVE

```

Comment:
Since this method fails when all elements are negative, while method 2 doesn't, it's better to have just method 2 and remove method 1. After all, method 2 is also Kadane's algorithm.